src/libs/actions/PaymentMethods.js

```diff
@@ -38,17 +38,35 @@ function getPaymentMethods() {
  * @param {Object} params
  */
 function addBillingCard(params) {
+    const cardYear = params.expirationDate.substr(3);
+    const cardMonth = params.expirationDate.substr(0, 2);
+
     API.AddBillingCard({
         cardNumber: params.cardNumber,
-        cardYear: params.expirationDate.substr(3),
-        cardMonth: params.expirationDate.substr(0, 2),
+        cardYear,
+        cardMonth,
         cardCVV: params.securityCode,
         addressName: params.nameOnCard,
         addressZip: params.zipCode,
         currency: CONST.CURRENCY.USD,
     }).then(((response) => {
         if (response.jsonCode === 200) {
-            Onyx.set(ONYXKEYS.CARD_LIST, response);
+            const cardObject = {
+                additionalData: {
+                    isBillingCard: true,
```

Comment:
I think these are backwards in this case, we don't want it to be their old expensify billing card, and it _is_ a p2p debit card.